### ISSUE TITLE
3.6.0b3 - issues

### ISSUE THREAD

**Issue by thezoggy** [sabnzbd/sabnzbd#2144]
seeing worse performance with b3 than b2 :( 
using windows binaries on win10

seeing the 'disk error' issue that others saw on earlier betas that I've never seen. also warning now when it fails to delete a par:
```
WARNING 2022-04-08 8:09pm Deleting \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol03-07.par2 failed!
```
```
2022-04-08 20:09:27,915::INFO::[notifier:123] Sending notification: Warning - Deleting \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol03-07.par2 failed! (type=warning, job_cat=None)
2022-04-08 20:09:27,915::WARNING::[newsunpack:1202] Deleting \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol03-07.par2 failed!
2022-04-08 20:09:27,916::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol03-07.par2
2022-04-08 20:09:27,916::WARNING::[newsunpack:1202] Deleting \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol03-07.par2 failed!
2022-04-08 20:09:27,916::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol00-01.par2
2022-04-08 20:09:27,917::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\zoggy\downloads\incomplete\Pivoting.S01E03.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\Pivoting.S01E03.The.Giving.Tree.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS.vol07-15.par2
```

--

```
ERROR 2022-04-08 8:10pm Disk error on creating file C:\Users\zoggy\downloads\incomplete\Pivoting.S01E05.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\6sCq93IoX4tt4rj1I0oFeg4q4eUhHWDQ6S4x8.part03.rar
```
```
2022-04-08 20:10:58,460::ERROR::[assembler:251] Disk error on creating file C:\Users\zoggy\downloads\incomplete\Pivoting.S01E05.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\6sCq93IoX4tt4rj1I0oFeg4q4eUhHWDQ6S4x8.part03.rar
2022-04-08 20:10:58,462::INFO::[assembler:254] Winerror: None - 0xc0000034
2022-04-08 20:10:58,462::INFO::[assembler:259] Traceback: 
Traceback (most recent call last):
  File "sabnzbd\assembler.py", line 184, in run
  File "sabnzbd\assembler.py", line 282, in assemble
PermissionError: [Errno 13] Permission denied: '\\\\?\\C:\\Users\\zoggy\\downloads\\incomplete\\Pivoting.S01E05.1080p.HULU.WEB-DL.DDP5.1.H.264-NPMS\\6sCq93IoX4tt4rj1I0oFeg4q4eUhHWDQ6S4x8.part03.rar'
2022-04-08 20:10:58,463::INFO::[downloader:393] Pausing
```

sent log file to bugs

**Comment by thezoggy** [comment]
downgraded back to 3.6.0b2 and d/led another 5 episodes and no warning/errors this time.

**Comment by Safihre** [comment]
Interesting. Could you try on Beta3 to set in Config Specials `assemblers` to 1? Does that fix the issues and restore performance?

**Comment by Safihre** [comment]
@jcfp Maybe not update the PPA until I fix this for next release. I reverted the update to `latest.txt` also.

**Comment by Safihre** [comment]
@thezoggy Do you maybe have any more logs? The logs you send only have 1 occurrence of the problem.

**Comment by thezoggy** [comment]
hey there, sorry I was out of pocket for the last few days. I only see the one winerror / "Disk error on creating file" as I noted. but I quickly reverted back to beta2 since I knew I was going to be gone and didnt want sab to keep pausing and stop working while I was away. Let me upgrade again and try some more downloads to see if its reproducible

**Comment by thezoggy** [comment]
ok upgraded back to 3.6.0b3

found a bug:
went to add test category and noticed the typeahead is not positioned correctly:
![firefox_2022-04-10_15-55-43](https://user-images.githubusercontent.com/500882/162639450-aaeaee94-f088-4924-bc7f-ecfe4399cb52.png)

**Comment by thezoggy** [comment]
loaded up a few nzbs manually, d/led just fine no warning/errors. performance was meh, decoders + assemblers  are both set to 0 via config>specials. changed assemblers to 1 and restarted sab. loaded up a few more nzbs, same performance but less cpu. but hard to really tell when downloads are completed within just a few seconds anyways. set watch folder to the folder i had some .nzb and let it load them that way, paused queue and set category to 'test' (feature idea? being able to set the category to use for the watch folder nzb additions). and those d/ls worked just fine as well. so far no error/warnings. 

looking back i do see that the downloads that triggered this were password protected, so let me go find some like those to test with

**Comment by thezoggy** [comment]
ok, started 3.6.0b3 again, assemblers 0. 

grabbed a season of a show thats pw protected, similar to what i saw before that triggered it.

added 3 episodes, one threw warnings cleaning up parset, but no errors
```
2022-04-10 17:41:11,763::INFO::[assembler:192] Decoding finished \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol03-07.par2
2022-04-10 17:41:11,763::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.par2
2022-04-10 17:41:11,763::DEBUG::[nzbstuff:463] Removing article database for SABnzbd_nzf_oethjo_w
2022-04-10 17:41:11,763::DEBUG::[filesystem:904] [sabnzbd\nzbstuff.py.remove_admin] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\__ADMIN__\SABnzbd_nzf_oethjo_w
2022-04-10 17:41:11,764::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol03-07.par2
2022-04-10 17:41:11,764::INFO::[notifier:123] Sending notification: Warning - Deleting \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol03-07.par2 failed! (type=warning, job_cat=None)
2022-04-10 17:41:11,764::WARNING::[newsunpack:1202] Deleting \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol03-07.par2 failed!
2022-04-10 17:41:11,765::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol03-07.par2
2022-04-10 17:41:11,765::WARNING::[newsunpack:1202] Deleting \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol03-07.par2 failed!
2022-04-10 17:41:11,766::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol00-01.par2
2022-04-10 17:41:11,767::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol01-03.par2
2022-04-10 17:41:11,768::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol07-15.par2
2022-04-10 17:41:11,773::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol15-31.par2
2022-04-10 17:41:11,776::DEBUG::[filesystem:904] [sabnzbd\newsunpack.py.par2_repair] Deleting file \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES.vol31-38.par2
2022-04-10 17:41:11,778::DEBUG::[filesystem:496] build_filelists(): joinables: []
2022-04-10 17:41:11,780::DEBUG::[filesystem:497] build_filelists(): zips: []
2022-04-10 17:41:11,780::DEBUG::[filesystem:498] build_filelists(): rars: ['\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part01.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part02.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part03.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part04.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part05.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part06.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part07.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part08.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part09.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part10.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part11.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part12.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part13.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part14.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part15.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part16.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part17.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part18.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part19.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part20.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part21.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part22.rar', '\\\\?\\C:\\Users\\<USERNAME>\\downloads\\incomplete\\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\\6bcqj6J9P61tp8K5DjM55ip616G1q66.part23.rar']
2022-04-10 17:41:11,780::DEBUG::[filesystem:499] build_filelists(): 7zips: []
2022-04-10 17:41:11,780::DEBUG::[filesystem:500] build_filelists(): ts: []
2022-04-10 17:41:11,780::DEBUG::[postproc:793] Verified sets: {'Chappelles.Show.S02E10.Mandela.Boot.Camp.and.The.Time.Haters.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES': True}
2022-04-10 17:41:11,781::DEBUG::[filesystem:1110] [sabnzbd\postproc.py.parring] Saving data for __verified__ in \\?\C:\Users\<USERNAME>\downloads\incomplete\Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES\__ADMIN__
2022-04-10 17:41:11,782::INFO::[postproc:796] Verification and repair finished for Chappelles.Show.S02E10.1080p.HMAX.WEB-DL.DD2.0.x264-TEPES
```

did another 3, no warnings/errors. 
changed assemblers to 1 and restarted sab

did another 3, no warnings/errors. 
enabled direct unpack. restarted sab

did another 3, no warnings/errors. 

so yeah, cant seem to reproduce.

**Comment by Safihre** [comment]
@thezoggy Could you give this build a go and see what happens?
https://github.com/sabnzbd/sabnzbd/actions/runs/2143534753
See how many results you get and which ones.

**Comment by thezoggy** [comment]
Will do

**Comment by thezoggy** [comment]
ok installed your newer build,
I noticed that specials > **num_simd_decoders** to 1 instead of the default 0. 

found a new show recently posted, pw protected. tossed first three episodes into the queue. all completed without warning/errors.

changed **num_simd_decoders**  to 0, restarted sab.
added three more episodes, all completed without warning/errors.

unchecked direct unpack, 
added three more episodes, all completed without warning/errors.

I noticed that when direct unpack is used, it does not show the "Trying unrar with password "xxxx"" line in the history under Unpack. Not sure if this is by design, but just to note in case its not:
```
Repair
[Thirty-Nine.S01E09.KOREAN.1080p.WEBRip.x265-RARBG] Quick Check OK
Unpack
[Thirty-Nine.S01E09.KOREAN.1080p.WEBRip.x265-RARBG.mp4] Trying unrar with password "5Z1cix2n1tjH"
[Thirty-Nine.S01E09.KOREAN.1080p.WEBRip.x265-RARBG.mp4] Unpacked 1 files/folders in 2 seconds
```
vs
```
Repair
[Thirty-Nine.S01E01.KOREAN.1080p.WEBRip.x265-RARBG] Quick Check OK
Unpack
[Thirty-Nine.S01E01.KOREAN.1080p.WEBRip.x265-RARBG.mp4] Direct Unpack - Unpacked 1 files/folders in 3 seconds
```

changed **num_simd_decoders**  to 1, restarted sab. (to test if directunpack not enable makes a difference)
added three more episodes, all completed without warning/errors.

so 12 episodes d/led with various settings.. no error/warnings.
